i18n/id.toml

```diff
@@ -0,0 +1,195 @@
+# Terjemahan untuk Bahasa Indonesia
+# https://gohugo.io/content-management/multilingual/#translation-of-strings
+
+# === baseof ==
+[backToTop]
+other = "Kembali ke Atas"
+
+[viewComments]
+other = "Lihat Komentar"
+# === baseof ==
+
+# === Post ===
+[posts]
+other = "Pos"
+# === Post ===
+
+# === Taxonomy ===
+[allSome]
+other = "Semua {{ .Some }}"
+
+[tag]
+other = "Label"
+
+[tags]
+other = "Label"
+
+[category]
+other = "Kategori"
+
+[categories]
+other = "Kategori"
+# === Taxonomy ===
+
+# === Pagination ===
+[more]
+other = "Lainnya"
+# === Pagination ===
+
+# === partials/header.html ===
+[selectLanguage]
+other = "Pilih Bahasa"
+
+[switchTheme]
+other = "Ganti Tema"
+# === partials/header.html ===
+
+# === partials/footer.html ===
+[poweredBySome]
+other = "Ditenagai oleh {{ .Hugo }} | Tema - {{ .Theme }}"
+# === partials/footer.html ===
+
+# === partials/comment.html ===
+[valineLang]
+other = "en"
+
+[valinePlaceholder]
+other = "Komentarmu ..."
+
+[facebookLanguageCode]
+other = "id"
+# === partials/comment.html ===
+
+# === partials/assets.html ===
+[search]
+other = "Pencarian"
+
+[searchPlaceholder]
+other = "Cari judul atau konten ..."
+
+[clear]
+other = "Bersihkan"
+
+[cancel]
+other = "Batal"
+
+[noResultsFound]
+other = "Hasil tidak ditemukan"
+
+[lunrLanguageCode]
+other = "en"
+
+[copyToClipboard]
+other = "Salin ke papan klip"
+
+[cookieconsentMessage]
+other = "Situs ini menggunakan Cookies untuk meningkatkan pengalaman Anda."
```

Comment:
`...anda.` not `...Anda.`

Reply:
In KBBI, word "Anda" should be capitalize.

Thank you for reviewing my PR. This is my first time contributing to someone else project. Once again, thank you :)